microprofile-opentracing/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,6 @@
+<beans xmlns="https://jakarta.ee/xml/ns/jakartaee"
```

Comment:
Is it necessary to have this file twice in the repo?

Reply:
you are right, I deleted it, re-run the tests, and it's the same: https://jenkins.eapqe.psi.redhat.com/job/eap-8.x-microprofile-testsuite/704/